SPECS/ca-certificates/ca-certificates.spec

```diff
@@ -363,6 +371,21 @@ rm -f %{pkidir}/tls/certs/*.{0,pem}
 %files shared
 %license LICENSE
 
+%files static
+# base files
+%{_datadir}/pki/ca-trust-source/%{p11_format_base_bundle}
+%{_datadir}/pki/ca-trust-legacy/%{legacy_default_base_bundle}
+%{_datadir}/pki/ca-trust-legacy/%{legacy_disable_base_bundle}
+# main pkg files
+%{_datadir}/pki/ca-trust-source/%{p11_format_mozilla_bundle}
+%{_datadir}/pki/ca-trust-legacy/%{legacy_default_mozilla_bundle}
+%{_datadir}/pki/ca-trust-legacy/%{legacy_disable_mozilla_bundle}
+# microsoft files
+%{_datadir}/pki/ca-trust-source/%{p11_format_microsoft_bundle}
+%{_datadir}/pki/ca-trust-legacy/%{legacy_default_microsoft_bundle}
+%{_datadir}/pki/ca-trust-legacy/%{legacy_disable_microsoft_bundle}
```

Comment:
Why is this change required? I believe you've mentioned it to me a few weeks ago, but I no longer recall. The `*-shared` subpackage installed all the directories and files required by subpackages actually installing the certificates. This seems now broken.